.travis.yml

```diff
@@ -31,20 +35,17 @@ jobs:
                 - echo "memory_limit=4096M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
                 - echo "extension = memcached.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 
-                - cp .env.test_cached.dist .env.test_cached
-                - set -a && source .env.test_cached && set +a
-
             install:
                 - if [ ! -z "${SYMFONY_VERSION}" ]; then bin/require-symfony-version composer.json "${SYMFONY_VERSION}"; fi
                 - composer update --no-interaction --prefer-dist
                 - yarn install
 
             before_script:
-                - bin/console doctrine:database:create --env=test_cached -vvv # Have to be run with debug = true, to omit generating proxies before setting up the database
-                - bin/console cache:warmup --env=test_cached --no-debug -vvv
-                - bin/console doctrine:migrations:migrate --no-interaction --env=test_cached --no-debug -vvv
+                - APP_DEBUG=1 bin/console doctrine:database:create -vvv # Have to be run with debug = true, to omit generating proxies before setting up the database
+                - APP_DEBUG=1 APP_ENV=dev bin/console cache:warmup -vvv
```

Comment:
I think we don't need `APP_DEBUG=1 APP_ENV=dev` here as it was run with `test_cached` and debug disabled before.